app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+from shutil import copy
+
+
+def copy_file(command: str) -> None:
+    _, source_file, destination_file = command.split(" ")
```

Comment:
Code Style Issue: It's recommended to use more descriptive variable names. The variable 'command' could be named 'copy_command' to better reflect its purpose.